src/components/KInput/KInput.vue

```diff
@@ -260,6 +263,21 @@ watch(() => props.error, (newVal, oldVal) => {
     helpTextKey.value += 1
   }
 })
+
+const beforeSlotElement = ref<HTMLElement>()
+const afterSlotElement = ref<HTMLElement>()
```

Comment:
nit: should be initialized as `null`
```suggestion
const beforeSlotElement = ref<HTMLElement | null>(null)
const afterSlotElement = ref<HTMLElement | null>(null)
```